docs/tre-admins/upgrading-tre.md

```diff
@@ -1,33 +1,35 @@
 # Upgrading AzureTRE version
 
-This document will cover how AzureTRE is referenced and how to upgrade its version in the [AzureTRE deployment repository](https://github.com/microsoft/AzureTRE-Deployment)
+This document will cover how Azure TRE is referenced and how to upgrade its version in the [Azure TRE deployment repository](https://github.com/microsoft/AzureTRE-Deployment)
 
 ## Introduction
 
-AzureTRE referenced as an external folder in [AzureTRE deployment repository](https://github.com/microsoft/AzureTRE-Deployment) (which is used as a template for your project in the quick start guide). A specific version of AzureTRE is downloaded as part of devcontainer setup.
+Azure TRE is referenced as an external folder in the [Azure TRE deployment repository](https://github.com/Microsoft/AzureTRE-Deployment) (which is used as a template for your project in the quick start guide). A specific version of Azure TRE is downloaded as part of the devcontainer setup.
 
 A symlink is then created making it available to reference in the directory itself (it is available only for reference, any changes to it are gitignored).
 
-## How to upgrade AzureTRE version
+## How to upgrade the Azure TRE version
 
 > Please check the release notes before upgrading.
 
-- If using the [AzureTRE deployment repository](https://github.com/microsoft/AzureTRE-Deployment) directly (not one created using a Template), you need to git pull the latest version.
+- If using the [Azure TRE deployment repository](https://github.com/microsoft/AzureTRE-Deployment) directly (not one created using a Template), you need to git pull the latest version.
 
-- If using a repository created from the `AzureTRE-Deployment` template then run following git command in your own repo:
+- If using a repository created from the `AzureTRE-Deployment` template, then run the following git commands in your own repo:
 ```sh
-git remote add upstream https://github.com/microsoft/AzureTRE-Deployment
+git remote add upstream https://github.com/Microsoft/AzureTRE-Deployment
 git pull upstream main --allow-unrelated-histories
 ```
-This will pull the latest version of AzureTRE to your copy of the repository. You may need to resovle merge conflicts if you have made edits.
+This will pull the latest version of AzureTRE to your copy of the repository. You may need to resolve merge conflicts if you have made edits.
```

Comment:
You missed this instance of `AzureTRE` 
```suggestion
This will pull the latest version of Azure TRE to your copy of the repository. You may need to resolve merge conflicts if you have made edits.
```